.github/workflows/release.yaml

```diff
@@ -72,6 +72,11 @@ jobs:
       - name: Generate Release Artifacts
         run: IMAGE_PULL_POLICY=IfNotPresent make generate-artifacts IMAGE=envoyproxy/gateway TAG=${{ env.release_tag }} OUTPUT_DIR=release-artifacts
 
+      - name: Generate and Package CRDs
+        run: |
+          make manifests
+          cd charts/gateway-helm/crds/ && zip -r envoy-gateway-crds.zip generated/
```

Comment:
why we need this?

Reply:
Since helm can't handle CRD upgrades, users can have the option to apply the yaml trough, for example kustomize instead of having to manually clone this repo and grab the CRDs from the helm chart.

So this will especially help users that already got the current helm chart installed and don't feel like changing over to the new one.